### PR TITLE
Fix the issue related to selectBox when option is empty string

### PR DESCRIPTION
**Pull request by ihappyk**
Fix for https://github.tools.sap/sProcurement/qmate/issues/328

### CODE REVIEW

**Review by heraldantony — DISMISSED** [review]
looks good

File: src/reuse/modules/ui5/userInteraction.ts [hunk 1]
```diff
@@ -427,7 +427,7 @@ export class UserInteraction {
   async selectBox(selector: any, value: string, index = 0) {
     const vl = this.vlf.initLog(this.selectBox);
     await this.clickSelectArrow(selector, index);
-    if (value) {
+    if (value !== undefined) {
```
Comment (DIFSRIP) [inline src/reuse/modules/ui5/userInteraction.ts:430]:
```suggestion
    if (value !== undefined && value !== null) {
```

**Review by DIFSRIP — CHANGES_REQUESTED** [review]
please see my suggestion

**Review by DIFSRIP — APPROVED** [review]
looks good

**Pull request merged**